app/assess/routes.py

```diff
@@ -138,3 +139,49 @@ def fund_round(fund_id: str, round_id: str):
         abort(404)
 
     return render_template("round.html", fund=fund, round=fund_round)
+
+
+@assess_bp.route("/fragments/text_input/")
+def text_input():
+    """
+    Render html page with json contains question & answer.
+    """
+
+    input_text_name = {
+        "questions": [
+            {
+                "fields": [
+                    {
+                        "key": "oLfixk",
+                        "title": "Your name",
+                        "type": "text",
+                        "answer": "Steve",
+                    },
+                ],
+            }
+        ],
+    }
+
+    input_text_address = {
+        "questions": [
+            {
+                "fields": [
+                    {
+                        "key": "gOgMvi",
+                        "title": "Your UK address",
+                        "type": "text",
+                        "answer": "99 evoco, example street, London, UB5 5FF",
+                    },
+                ],
+            }
+        ],
+    }
+
+    name = QuestionField.get_question_fields(input_text_name)
+    address = QuestionField.get_question_fields(input_text_address)
```

Comment:
Instead of this section I think you could just do 
name = QuestionField(key="abc",title="Your name",field_type="text",answer="Steve")
address = QuestionField(key="xyz",title="Your UK address",field_type="text",answer="99 evoco, example street, London, UB5 5FF")
As there's no need to reprocess json data with a separate function (get_question_fields) as from_json already does that

Reply:
I used the from_json function instead 😄